UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -436,7 +497,9 @@
     gEfiMdePkgTokenSpaceGuid.PcdDebugPropertyMask|0x03
   !endif
 !endif
-
+  gEfiMdeModulePkgTokenSpaceGuid.PcdMaxSizeNonPopulateCapsule|$(MAX_SIZE_NON_POPULATE_CAPSULE)
+  gPcAtChipsetPkgTokenSpaceGuid.PcdRtcIndexRegister|$(RTC_INDEX_REGISTER)
+  gPcAtChipsetPkgTokenSpaceGuid.PcdRtcTargetRegister|$(RTC_TARGET_REGISTER)

```

Comment:
Duplicates lines 478 and 479

Reply:
Fixed https://github.com/Dasharo/edk2/pull/38/commits/831e24bdc537499438634a9a088b7eb5f2eacb30